dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/dto/resources/ResourceComponent.java

```diff
@@ -1,7 +1,7 @@
 package org.apache.dolphinscheduler.api.dto.resources;
 
-import com.alibaba.fastjson.annotation.JSONField;
 import com.alibaba.fastjson.annotation.JSONType;
+import com.alibaba.fastjson.annotation.JSONField;
```

Comment:
why still using the fastjson annotation?

Reply:
sorry，I missed it 
i will fix it later